pages/offices/index.tsx

```diff
@@ -141,6 +141,13 @@ export const getStaticProps = async () => {
   const testimonial =
     testimonials[Math.floor(Math.random() * testimonials.length)].Testimonial;
 
+  if (
+    tinaProps.data.officeIndex.seo &&
+    !tinaProps.data.officeIndex.seo.canonical
+  ) {
+    tinaProps.data.officeIndex.seo.canonical = "/offices";
```

Comment:
I think these need to be fully qualified in case there are differences in the base URL, eg http vs https
Same for consulting and products

Reply:
Done, had to add extra query for consulting and products as they current operate by making a `connection` query